backend/src/test/java/com/woowacourse/f12/presentation/product/CategoryConstantTest.java

```diff
@@ -0,0 +1,60 @@
+package com.woowacourse.f12.presentation.product;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.woowacourse.f12.domain.product.Category;
+import com.woowacourse.f12.exception.badrequest.InvalidCategoryValueException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class CategoryConstantTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {"keyboard:KEYBOARD", "software:SOFTWARE"}, delimiter = ':')
```

Comment:
```suggestion
    @ParameterizedTest
    @CsvSource(value = {"keyboard,KEYBOARD", "software,SOFTWARE"})
```
구분자를 지정하지 않으면 자동으로 `,`로 구분합니다.

Reply:
반영했습니다.